src/rm.js

```diff
@@ -106,43 +116,53 @@ function handleFile(file, options) {
 function handleDirectory(file, options) {
   if (options.recursive) {
     // -r was passed, so directory can be removed
-    rmdirSyncRecursive(file, options.force);
+    rmdirSyncRecursive(file, options.force, options.verbose);
   } else {
     common.error('path is a directory', { continue: true });
   }
 }
 
 function handleSymbolicLink(file, options) {
+  function _unlink() {
+    common.unlinkSync(file);
+    if (options.verbose) {
+      console.log("removed '" + file + "'");
+    }
```

Comment:
Can you change this so that `file` is a method parameter? I'd also suggest passing `options` as a second parameter and then moving this function definition to the global scope, similar to what you've done with `handleFIFO()`.